### PR TITLE
Community PR - Dune Dashboard (2a)

### PR DESCRIPTION
**Pull request by mkchungs**
# Community PR - Dune Dashboard Phase2a

This is the followup PR of #5933, where we have included another 20+ parachains & dashboards from Dune Integration Order Form 2.

**Documentation Previews**

Onboarding-Status-Key-Dashboards-·-Polkadot-Wiki

Polkadot-Ecosystem-Overview-·-Polkadot-Wiki

### CODE REVIEW

File: docs/general/dune-analytics/polkadot-dashboards/polkadot-dashboards-governance.md [hunk 1]
```diff
@@ -3,10 +3,8 @@ id: polkadot-dashboards-governance
 title: "Polkadot Dashboards: Governance"
 sidebar_label: Governance
 description:
-  This document demonstrated various Polkadot governance dashboards, offering insights into treasury
-  management, referendum outcomes, voting patterns, and OpenGov referenda. These dashboards aim to
-  enhance community engagement by simplifying access to governance information and fostering a
-  transparent decision-making environment.
+  An comprehensive data analysis of Polkadot governance systems, including treasury management,
```
Comment (filippoweb3) [inline docs/general/dune-analytics/polkadot-dashboards/polkadot-dashboards-governance.md]:
```suggestion
  A comprehensive data analysis of Polkadot governance systems, including treasury management,
```

File: docs/general/dune-analytics/parachain-dashboards/snowbridge-dashboards.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+- `bridgehub.blocks`
+- `bridgehub.calls`
+- `bridgehub.events`,
+- `bridgehub.extrinsics`
+- `bridgehub.transfers`
+
+## Useful Queries
+
+Currently, there are no specific useful queries provided. Please check back later as this section
+will be updated with materialized queries for Snowbridge.
```
Comment (filippoweb3) [inline docs/general/dune-analytics/parachain-dashboards/snowbridge-dashboards.md]:
```suggestion
Currently, there are no specific queries provided.
```

File: docs/general/dune-analytics/parachain-dashboards/astar-dashboards.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
+---
+id: astar-dashboards
+title: Astar Dashboards
+sidebar_label: Astar
+description:
+  Astar Network, previously known as Plasm, is a decentralized blockchain platform designed for
+  Web3. It is compatible with both EVM (Ethereum Virtual Machine) and WebAssembly, enabling seamless
+  interaction between the two environments. Developers can participate in the Build2Earn program to
+  earn rewards by creating decentralized applications.
```
Comment (filippoweb3) [inline docs/general/dune-analytics/parachain-dashboards/astar-dashboards.md]:
```suggestion
  Astar Network is a blockchain platform designed for
  Web3. It is compatible with both EVM (Ethereum Virtual Machine) and WebAssembly, enabling seamless
  interaction between the two environments.
```

File: docs/general/dune-analytics/parachain-dashboards/astar-dashboards.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+---
+
+# Astar Dashboards
+
+## Overview
+
+Astar Network, previously known as Plasm, is a decentralized blockchain platform designed for Web3.
+It is compatible with both EVM (Ethereum Virtual Machine) and WebAssembly, enabling seamless
+interaction between the two environments. Developers can participate in the Build2Earn program to
+earn rewards by creating decentralized applications.
```
Comment (filippoweb3) [inline docs/general/dune-analytics/parachain-dashboards/astar-dashboards.md]:
```suggestion
Astar Network, previously known as Plasm, is a blockchain platform designed for Web3.
It is compatible with both EVM (Ethereum Virtual Machine) and WebAssembly, enabling seamless
interaction between the two environments. Developers can participate in the Build2Earn program to
earn rewards by creating decentralized applications.
```

File: docs/general/dune-analytics/parachain-dashboards/astar-dashboards.md [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+- `astar.balances`
+- `astar.blocks`
+- `astar.calls`
+- `astar.events`,
+- `astar.extrinsics`
+- `astar.transfers`
+
+## Useful Queries
+
+Currently, there are no specific queries provided. Please check back later for updates.
```
Comment (filippoweb3) [inline docs/general/dune-analytics/parachain-dashboards/astar-dashboards.md]:
```suggestion
Currently, there are no specific queries provided.
```

File: docs/general/dune-analytics/parachain-dashboards/bridgehub-dashboards.md [hunk 6]
```diff
@@ -0,0 +1,153 @@
+---
+id: bridgehub-dashboards
+title: BridgeHub Dashboards
+sidebar_label: BridgeHub
+description: Polkadot BridgeHub is a system parachain within the Polkadot ecosystem, designed to
+  enable trustless bridging between Polkadot and other blockchains such as Kusama and Ethereum. It
+  ensures secure cross-chain communication through a combination of on-chain and off-chain
+  components. This dashboard primarily analyzes the bridging information between Polkadot and
+  Kusama.
```
Comment (filippoweb3) [inline docs/general/dune-analytics/parachain-dashboards/bridgehub-dashboards.md]:
```suggestion
description: Polkadot BridgeHub is a system parachain within the Polkadot ecosystem, designed to
  enable trustless bridging between Polkadot and other blockchains such as Kusama and Ethereum.
```

File: docs/general/dune-analytics/parachain-dashboards/bridgehub-dashboards.md [hunk 7]
```diff
@@ -0,0 +1,153 @@
...
+- `bridgehub.blocks`
+- `bridgehub.calls`
+- `bridgehub.events`,
+- `bridgehub.extrinsics`
+- `bridgehub.transfers`
+
+## Useful Queries
+
+Currently, there are no specific useful queries provided. Please check back later as this section
+will be updated with materialized queries for BridgeHub.
```
Comment (filippoweb3) [inline docs/general/dune-analytics/parachain-dashboards/bridgehub-dashboards.md]:
```suggestion
Currently, there are no specific queries provided.
```

File: docs/general/dune-analytics/parachain-dashboards/centrifuge-dashboards.md [hunk 8]
```diff
@@ -0,0 +1,139 @@
...
+
+Centrifuge's ecosystem extends beyond its onchain financial infrastructure, incorporating a DAO
+(decentralized autonomous organization) supported by a diverse community of finance professionals
+and developers.
+
+By bringing the entire structured credit market onchain across securitization, tokenization,
+privacy, governance, and liquidity integrations, Centrifuge is building a more transparent,
+affordable, and limitless financial system.
+
+Some assets are managed on Ethereum, others are managed on Centrifuge (a polkadot parachain.)
```
Comment (filippoweb3) [inline docs/general/dune-analytics/parachain-dashboards/centrifuge-dashboards.md]:
```suggestion
Some assets are managed on Ethereum, others are managed on Centrifuge.
```

File: docs/general/dune-analytics/parachain-dashboards/collectives-dashboards.md [hunk 9]
```diff
@@ -0,0 +1,86 @@
+---
+id: collectives-dashboards
+title: Collectives Dashboards
+sidebar_label: Collectives
+description:
+  The Technical Fellowship is a self-governing body of experts and developers of Polkadot and Kusama
+  networks protocols. It operates on-chain through the Polkadot Collectives system chain and
+  off-chain through the Polkadot Fellows repository.
```
Comment (filippoweb3) [inline docs/general/dune-analytics/parachain-dashboards/collectives-dashboards.md]:
```suggestion
description:
  The Technical Fellowship is a self-governing body of experts and developers of Polkadot and Kusama
  networks protocols.
```

File: docs/general/dune-analytics/parachain-dashboards/collectives-dashboards.md [hunk 10]
```diff
@@ -0,0 +1,86 @@
...
+
+keywords: [polkadot, dashboard, dune, collectives, fellowship]
+slug: ../collectives-dashboards
+---
+
+# Manta Dashboards
```
Comment (filippoweb3) [inline docs/general/dune-analytics/parachain-dashboards/collectives-dashboards.md]:
```suggestion
# Collectives Dashboards
```

File: docs/general/dune-analytics/parachain-dashboards/collectives-dashboards.md [hunk 11]
```diff
@@ -0,0 +1,86 @@
...
+
+## Overview
+
+The Technical Fellowship is a self-governing body of experts and developers of Polkadot and Kusama
+networks protocols. It operates on-chain through the Polkadot Collectives system chain and off-chain
+through the Polkadot Fellows repository.
+
+## Featured Dashboards on Dune
+
+Here you'll find a variety of dashboards that help visualize data from the Manta parachain:
```
Comment (filippoweb3) [inline docs/general/dune-analytics/parachain-dashboards/collectives-dashboards.md]:
```suggestion
Here you will find a variety of dashboards that help visualize data from the Collectives parachain:
```

File: docs/general/dune-analytics/parachain-dashboards/collectives-dashboards.md [hunk 12]
```diff
@@ -0,0 +1,86 @@
...
+- `collectives.balances`
+- `collectives.blocks`
+- `collectives.calls`
+- `collectives.events`,
+- `collectives.extrinsics`
+- `collectives.transfers`
+
+## Useful Queries
+
+Currently, there are no specific queries provided. Please check back later for updates.
```
Comment (filippoweb3) [inline docs/general/dune-analytics/parachain-dashboards/collectives-dashboards.md]:
```suggestion
Currently, there are no specific queries provided.
```

File: docs/general/dune-analytics/parachain-dashboards/energywebx-dashboards.md [hunk 13]
```diff
@@ -0,0 +1,102 @@
...
+- `energywebx.balances`
+- `energywebx.blocks`
+- `energywebx.calls`
+- `energywebx.events`,
+- `energywebx.extrinsics`
+- `energywebx.transfers`
+
+## Useful Queries
+
+Currently, there are no specific queries provided. Please check back later for updates.
```
Comment (filippoweb3) [inline docs/general/dune-analytics/parachain-dashboards/energywebx-dashboards.md]:
```suggestion
Currently, there are no specific queries provided.
```

File: docs/general/dune-analytics/parachain-dashboards/frequency-dashboards.md [hunk 14]
```diff
@@ -0,0 +1,78 @@
...
+
+## Overview
+
+Frequency is a parachain on Polkadot focusing on decentralized communication solutions. It allows
+the creation and management of decentralized networks for various applications, enhancing data
+privacy and security.
+
+## Featured Dashboards on Dune
+
+Here you'll find a variety of dashboards that help visualize data from the Frequency parachain:
```
Comment (filippoweb3) [inline docs/general/dune-analytics/parachain-dashboards/frequency-dashboards.md]:
```suggestion
Here you will find a variety of dashboards that help visualize data from the Frequency parachain:
```

File: docs/general/dune-analytics/parachain-dashboards/frequency-dashboards.md [hunk 15]
```diff
@@ -0,0 +1,78 @@
...
+- `frequency.balances`
+- `frequency.blocks`
+- `frequency.calls`
+- `frequency.events`,
+- `frequency.extrinsics`
+- `frequency.transfers`
+
+## Useful Queries
+
+Currently, there are no specific queries provided. Please check back later for updates.
```
Comment (filippoweb3) [inline docs/general/dune-analytics/parachain-dashboards/frequency-dashboards.md]:
```suggestion
Currently, there are no specific queries provided.
```

File: docs/general/dune-analytics/parachain-dashboards/invarch-dashboards.md [hunk 16]
```diff
@@ -0,0 +1,86 @@
...
+
+The InvArch Network is a Multichain Account abstraction hub, optimized for decentralized operations
+& financial management spanning across every blockchain.
+
+InvArch provides individuals, organizations & DAOs with a single account that can be used to manage
+all of their assets & execute transactions across any network.
+
+## Featured Dashboards on Dune
+
+Here you'll find a variety of dashboards that help visualize data from the Acala parachain:
```
Comment (filippoweb3) [inline docs/general/dune-analytics/parachain-dashboards/invarch-dashboards.md]:
```suggestion
Here you will find a variety of dashboards that help visualize data from the Acala parachain:
```

File: docs/general/dune-analytics/parachain-dashboards/invarch-dashboards.md [hunk 17]
```diff
@@ -0,0 +1,86 @@
...
+- `invarch.balances`
+- `invarch.blocks`
+- `invarch.calls`
+- `invarch.events`,
+- `invarch.extrinsics`
+- `invarch.transfers`
+
+## Useful Queries
+
+Currently, there are no specific queries provided. Please check back later for updates.
```
Comment (filippoweb3) [inline docs/general/dune-analytics/parachain-dashboards/invarch-dashboards.md]:
```suggestion
Currently, there are no specific queries provided.
```

File: docs/general/dune-analytics/parachain-dashboards/litentry-dashboards.md [hunk 18]
```diff
@@ -0,0 +1,100 @@
+---
+id: litentry-dashboards
+title: Litentry Dashboards
+sidebar_label: Litentry
+description:
+  Litentry is a decentralized identity aggregator that links user identities while ensuring privacy
+  protection, thus enabling various social and economic innovations. It features a three-layer
+  structure, comprising source data, address analysis, and identity aggregation layers, all secured
+  with Trusted Execution Environment (TEE) technology to enhance security and privacy. Litentry's
+  applications include airdrop whitelisting, credit scoring, and cross-platform reputation building.
```
Comment (filippoweb3) [inline docs/general/dune-analytics/parachain-dashboards/litentry-dashboards.md]:
```suggestion
description:
  Litentry is a decentralized identity aggregator that links user identities while ensuring privacy
  protection.
```

File: docs/general/dune-analytics/parachain-dashboards/litentry-dashboards.md [hunk 19]
```diff
@@ -0,0 +1,100 @@
...
+
+Litentry is a decentralized identity aggregator that links user identities while ensuring privacy
+protection, thus enabling various social and economic innovations. It features a three-layer
+structure, comprising source data, address analysis, and identity aggregation layers, all secured
+with Trusted Execution Environment (TEE) technology to enhance security and privacy. Litentry's
+applications include airdrop whitelisting, credit scoring, and cross-platform reputation building.
+
+## Featured Dashboards on Dune
+
+Here you'll find a variety of dashboards that help visualize data from the Litentry parachain:
```
Comment (filippoweb3) [inline docs/general/dune-analytics/parachain-dashboards/litentry-dashboards.md]:
```suggestion
Here you will find a variety of dashboards that help visualize data from the Litentry parachain:
```

File: docs/general/dune-analytics/parachain-dashboards/litentry-dashboards.md [hunk 20]
```diff
@@ -0,0 +1,100 @@
...
+- `litentry.blocks`
+- `litentry.calls`
+- `litentry.events`,
+- `litentry.extrinsics`
+- `litentry.transfers`
+
+## Useful Queries
+
+Currently, there are no specific useful queries provided. Please check back later as this section
+will be updated with materialized queries for Litentry.
```
Comment (filippoweb3) [inline docs/general/dune-analytics/parachain-dashboards/litentry-dashboards.md]:
```suggestion
Currently, there are no specific queries provided. 
```

File: docs/general/dune-analytics/parachain-dashboards/manta-dashboards.md [hunk 21]
```diff
@@ -0,0 +1,77 @@
+---
+id: manta-dashboards
+title: Manta Dashboards
+sidebar_label: Manta
+description:
+  Manta Atlantic is the fastest and most decentralized ZK Layer1 supporting modular on-chain
+  compliance identities. Its suite of core products and technologies, including zkNFTs, staking and
+  ecosystem projects, offers user-friendly access to powerful ZK-enabled use cases.
```
Comment (filippoweb3) [inline docs/general/dune-analytics/parachain-dashboards/manta-dashboards.md]:
```suggestion
  Manta Atlantic is a fast and decentralized ZK Layer 1 supporting modular on-chain
  compliance identities.
```

File: docs/general/dune-analytics/parachain-dashboards/manta-dashboards.md [hunk 22]
```diff
@@ -0,0 +1,77 @@
...
+slug: ../manta-dashboards
+---
+
+# Manta Dashboards
+
+## Overview
+
+Manta Atlantic is the fastest and most decentralized ZK Layer1 supporting modular on-chain
+compliance identities. Its suite of core products and technologies, including zkNFTs, staking and
+ecosystem projects, offers user-friendly access to powerful ZK-enabled use cases.
```
Comment (filippoweb3) [inline docs/general/dune-analytics/parachain-dashboards/manta-dashboards.md]:
```suggestion
Manta Atlantic is a fast and decentralized ZK Layer 1 supporting modular on-chain
compliance identities. Its suite of core products and technologies, including zkNFTs, staking, and
ecosystem projects, offers user-friendly access to powerful ZK-enabled use cases.
```

File: docs/general/dune-analytics/parachain-dashboards/manta-dashboards.md [hunk 23]
```diff
@@ -0,0 +1,77 @@
...
+
+## Featured Dashboards on Dune
+
+Here you'll find a variety of dashboards that help visualize data from the Manta parachain:
```
Comment (filippoweb3) [inline docs/general/dune-analytics/parachain-dashboards/manta-dashboards.md]:
```suggestion
Here you will find a variety of dashboards that help visualize data from the Manta parachain:
```

File: docs/general/dune-analytics/parachain-dashboards/manta-dashboards.md [hunk 24]
```diff
@@ -0,0 +1,77 @@
...
+- `manta.balances`
+- `manta.blocks`
+- `manta.calls`
+- `manta.events`,
+- `manta.extrinsics`
+- `manta.transfers`
+
+## Useful Queries
+
+Currently, there are no specific queries provided. Please check back later for updates.
```
Comment (filippoweb3) [inline docs/general/dune-analytics/parachain-dashboards/manta-dashboards.md]:
```suggestion
Currently, there are no specific queries provided.
```

File: docs/general/dune-analytics/parachain-dashboards/neuroweb-dashboards.md [hunk 25]
```diff
@@ -0,0 +1,87 @@
+---
+id: neuroweb-dashboards
+title: Neuroweb Dashboards
+sidebar_label: Neuroweb
+description:
+  NeuroWeb is a decentralized AI blockchain that rewards knowledge creation and sharing. Its NEURO
+  token supports the AI economy by incentivizing contributions to the OriginTrail Decentralized
+  Knowledge Graph (DKG). NeuroWeb builds upon the groundwork established by its predecessor, the
+  OriginTrail Parachain (OTP). This transformation into NeuroWeb was facilitated through a community
+  governance vote on OTP in December 2023.
```
Comment (filippoweb3) [inline docs/general/dune-analytics/parachain-dashboards/neuroweb-dashboards.md]:
```suggestion
  NeuroWeb is a decentralized AI blockchain that rewards knowledge creation and sharing.
```

File: docs/general/dune-analytics/parachain-dashboards/neuroweb-dashboards.md [hunk 26]
```diff
@@ -0,0 +1,87 @@
...
+
+NeuroWeb is a decentralized AI blockchain that rewards knowledge creation and sharing. Its NEURO
+token supports the AI economy by incentivizing contributions to the OriginTrail Decentralized
+Knowledge Graph (DKG). NeuroWeb builds upon the groundwork established by its predecessor, the
+OriginTrail Parachain (OTP). This transformation into NeuroWeb was facilitated through a community
+governance vote on OTP in December 2023.
+
+## Featured Dashboards on Dune
+
+Here you'll find a variety of dashboards that help visualize data from the Neuroweb parachain:
```
Comment (filippoweb3) [inline docs/general/dune-analytics/parachain-dashboards/neuroweb-dashboards.md]:
```suggestion
Here you will find a variety of dashboards that help visualize data from the Neuroweb parachain:
```

File: docs/general/dune-analytics/parachain-dashboards/neuroweb-dashboards.md [hunk 27]
```diff
@@ -0,0 +1,87 @@
...
+- `neuroweb.blocks`
+- `neuroweb.calls`
+- `neuroweb.events`,
+- `neuroweb.extrinsics`
+- `neuroweb.transfers`
+
+## Useful Queries
+
+Currently, there are no specific useful queries provided. Please check back later as this section
+will be updated with materialized queries for Neuroweb.
```
Comment (filippoweb3) [inline docs/general/dune-analytics/parachain-dashboards/neuroweb-dashboards.md]:
```suggestion
Currently, there are no specific queries provided.
```

File: docs/general/dune-analytics/parachain-dashboards/nodle-dashboards.md [hunk 28]
```diff
@@ -0,0 +1,82 @@
+---
+id: nodle-dashboards
+title: Nodle Dashboards
+sidebar_label: Nodle
+description:
+  Nodle is a parachain on Polkadot focused on providing connectivity and data liquidity for the
+  Internet of Things (IoT). It enables devices to securely connect, interact, and transact with
+  efficiency.
```
Comment (filippoweb3) [inline docs/general/dune-analytics/parachain-dashboards/nodle-dashboards.md]:
```suggestion
  Nodle is a parachain on Polkadot focused on providing connectivity and data liquidity for the
  Internet of Things (IoT).
```

File: docs/general/dune-analytics/parachain-dashboards/nodle-dashboards.md [hunk 29]
```diff
@@ -0,0 +1,82 @@
...
+
+## Overview
+
+Nodle is a parachain on Polkadot focused on providing connectivity and data liquidity for the
+Internet of Things (IoT). It enables devices to securely connect, interact, and transact with
+efficiency.
+
+## Featured Dashboards on Dune
+
+Here you'll find a variety of dashboards that help visualize data from the Nodle parachain:
```
Comment (filippoweb3) [inline docs/general/dune-analytics/parachain-dashboards/nodle-dashboards.md]:
```suggestion
Here you will find a variety of dashboards that help visualize data from the Nodle parachain:
```

File: docs/general/dune-analytics/parachain-dashboards/nodle-dashboards.md [hunk 30]
```diff
@@ -0,0 +1,82 @@
...
+- `nodle.balances`
+- `nodle.blocks`
+- `nodle.calls`
+- `nodle.events`,
+- `nodle.extrinsics`
+- `nodle.transfers`
+
+## Useful Queries
+
+Currently, no specific queries have been provided. Please check back later for updates.
```
Comment (filippoweb3) [inline docs/general/dune-analytics/parachain-dashboards/nodle-dashboards.md]:
```suggestion
Currently, no specific queries have been provided.
```

File: docs/general/dune-analytics/parachain-dashboards/pendulum-dashboards.md [hunk 31]
```diff
@@ -0,0 +1,76 @@
+---
+id: pendulum-dashboards
+title: Pendulum Dashboards
+sidebar_label: Pendulum
+description:
+  Pendulum is a parachain on Polkadot that focuses on bridging fiat currencies and decentralized
+  finance (DeFi). It aims to create a fully functional fiat-optimized blockchain that facilitates
+  open financial applications and connects them with the traditional financial sector.
```
Comment (filippoweb3) [inline docs/general/dune-analytics/parachain-dashboards/pendulum-dashboards.md]:
```suggestion
  Pendulum focuses on bridging fiat currencies and decentralized
  finance (DeFi).
```

File: docs/general/dune-analytics/parachain-dashboards/pendulum-dashboards.md [hunk 32]
```diff
@@ -0,0 +1,76 @@
...
+
+## Overview
+
+Pendulum is a parachain on Polkadot that focuses on bridging fiat currencies and decentralized
+finance (DeFi). It aims to create a fully functional fiat-optimized blockchain that facilitates open
+financial applications and connects them with the traditional financial sector.
+
+## Featured Dashboards on Dune
+
+Here you'll find a variety of dashboards that help visualize data from the Pendulum parachain:
```
Comment (filippoweb3) [inline docs/general/dune-analytics/parachain-dashboards/pendulum-dashboards.md]:
```suggestion
Here you will find a variety of dashboards that help visualize data from the Pendulum parachain:
```

File: docs/general/dune-analytics/parachain-dashboards/people-dashboards.md [hunk 33]
```diff
@@ -0,0 +1,84 @@
+---
+id: people-dashboards
+title: People Dashboards
+sidebar_label: People
+description:
+  People is a parachain on Polkadot focused on decentralized identity and social interactions,
+  enabling users to manage their digital identity and engage in community governance.
```
Comment (filippoweb3) [inline docs/general/dune-analytics/parachain-dashboards/people-dashboards.md]:
```suggestion
  People's Chain focuses on decentralized identity and social interactions,
  enabling users to manage their digital identity and engage in community governance.
```

File: docs/general/dune-analytics/parachain-dashboards/people-dashboards.md [hunk 34]
```diff
@@ -0,0 +1,84 @@
...
+keywords: [polkadot, dashboard, dune, people, identity]
+slug: ../people-dashboards
+---
+
+# People Dashboards
+
+## Overview
+
+People is a parachain on Polkadot focused on decentralized identity and social interactions,
+enabling users to manage their digital identity and engage in community governance.
```
Comment (filippoweb3) [inline docs/general/dune-analytics/parachain-dashboards/people-dashboards.md]:
```suggestion
People's Chain focuses on decentralized identity and social interactions,
enabling users to manage their digital identity and engage in community governance.
```

File: docs/general/dune-analytics/parachain-dashboards/people-dashboards.md [hunk 35]
```diff
@@ -0,0 +1,84 @@
...
+
+## Featured Dashboards on Dune
+
+Here you'll find a variety of dashboards that help visualize data from the People parachain:
```
Comment (filippoweb3) [inline docs/general/dune-analytics/parachain-dashboards/people-dashboards.md]:
```suggestion
Here you will find a variety of dashboards that help visualize data from the People parachain:
```

File: docs/general/dune-analytics/parachain-dashboards/phala-dashboards.md [hunk 36]
```diff
@@ -0,0 +1,82 @@
+---
+id: phala-dashboards
+title: Phala Dashboards
+sidebar_label: Phala
+description:
+  Phala is a parachain on Polkadot that provides a privacy-preserving computation and data
+  protection framework. It allows for the processing of sensitive data with guaranteed privacy.
```
Comment (filippoweb3) [inline docs/general/dune-analytics/parachain-dashboards/phala-dashboards.md]:
```suggestion
  Phala provides a privacy-preserving computation and data
  protection framework. 
```

File: docs/general/dune-analytics/parachain-dashboards/phala-dashboards.md [hunk 37]
```diff
@@ -0,0 +1,82 @@
...
+# Phala Dashboards
+
+## Overview
+
+Phala is a parachain on Polkadot that provides a privacy-preserving computation and data protection
+framework. It allows for the processing of sensitive data with guaranteed privacy.
+
+## Featured Dashboards on Dune
+
+Here you'll find a variety of dashboards that help visualize data from the Phala parachain:
```
Comment (filippoweb3) [inline docs/general/dune-analytics/parachain-dashboards/phala-dashboards.md]:
```suggestion
Here you will find a variety of dashboards that help visualize data from the Phala parachain:
```

File: docs/general/dune-analytics/parachain-dashboards/phala-dashboards.md [hunk 38]
```diff
@@ -0,0 +1,82 @@
...
+- `phala.calls`
+- `phala.events`,
+- `phala.extrinsics`
+- `phala.transfers`
+
+Additional curated dataset from Phala team can be found at `dune.phala_network.*`
+
+## Useful Queries
+
+Currently, no specific queries have been provided. Please check back later for updates.
```
Comment (filippoweb3) [inline docs/general/dune-analytics/parachain-dashboards/phala-dashboards.md]:
```suggestion
Currently, no specific queries have been provided.
```

File: docs/general/dune-analytics/parachain-dashboards/polimec-dashboards.md [hunk 39]
```diff
@@ -0,0 +1,106 @@
+---
+id: polimec-dashboards
+title: Polimec Dashboards
+sidebar_label: Polimec
+description:
+  Polimec offers a decentralized, transparent, and compliant method for fundraising, ensuring that
+  stakeholder incentives are aligned both during and after the fundraising process.
```
Comment (filippoweb3) [inline docs/general/dune-analytics/parachain-dashboards/polimec-dashboards.md]:
```suggestion
  Polimec offers a decentralized, transparent, and compliant method for fundraising.
```

File: docs/general/dune-analytics/parachain-dashboards/polimec-dashboards.md [hunk 40]
```diff
@@ -0,0 +1,106 @@
...
+# Polimec Dashboards
+
+## Overview
+
+Polimec offers a decentralized, transparent, and compliant method for fundraising, ensuring that
+stakeholder incentives are aligned both during and after the fundraising process.
+
+## Featured Dashboards on Dune
+
+Here you'll find a variety of dashboards that help visualize data from the Moonbeam parachain:
```
Comment (filippoweb3) [inline docs/general/dune-analytics/parachain-dashboards/polimec-dashboards.md]:
```suggestion
Here you will find a variety of dashboards that help visualize data from the Moonbeam parachain:
```

File: docs/general/dune-analytics/parachain-dashboards/polimec-dashboards.md [hunk 41]
```diff
@@ -0,0 +1,106 @@
...
+- `polimec.blocks`
+- `polimec.calls`
+- `polimec.events`,
+- `polimec.extrinsics`
+- `polimec.transfers`
+
+## Useful Queries
+
+Currently, there are no specific useful queries provided. Please check back later as this section
+will be updated with materialized queries for Neuroweb.
```
Comment (filippoweb3) [inline docs/general/dune-analytics/parachain-dashboards/polimec-dashboards.md]:
```suggestion
Currently, there are no specific queries provided.
```

File: docs/general/dune-analytics/parachain-dashboards/polkadex-dashboards.md [hunk 42]
```diff
@@ -0,0 +1,73 @@
+---
+id: polkadex-dashboards
+title: Polkadex Dashboards
+sidebar_label: Polkadex
+description:
+  Polkadex is a decentralized exchange parachain on Polkadot that combines the benefits of
+  centralized and decentralized exchanges, offering a high-performance trading experience with
+  trustless custody.
```
Comment (filippoweb3) [inline docs/general/dune-analytics/parachain-dashboards/polkadex-dashboards.md]:
```suggestion
  Polkadex is a decentralized exchange that combines the benefits of
  centralized and decentralized exchanges, offering a high-performance trading experience with
  trustless custody.
```

File: docs/general/dune-analytics/parachain-dashboards/polkadex-dashboards.md [hunk 43]
```diff
@@ -0,0 +1,73 @@
...
+## Overview
+
+Polkadex is a decentralized exchange parachain on Polkadot that combines the benefits of centralized
+and decentralized exchanges, offering a high-performance trading experience with trustless custody.
+It aims to provide a seamless trading environment while ensuring the security of a decentralized
+platform.
+
+## Featured Dashboards on Dune
+
+Here you'll find a variety of dashboards that help visualize data from the Polkadex parachain:
```
Comment (filippoweb3) [inline docs/general/dune-analytics/parachain-dashboards/polkadex-dashboards.md]:
```suggestion
Here you will find a variety of dashboards that help visualize data from the Polkadex parachain:
```

File: docs/general/dune-analytics/parachain-dashboards/polkadex-dashboards.md [hunk 44]
```diff
@@ -0,0 +1,73 @@
...
+- `polkadex.balances`
+- `polkadex.blocks`
+- `polkadex.calls`
+- `polkadex.events`,
+- `polkadex.extrinsics`
+- `polkadex.transfers`
+
+## Useful Queries
+
+Currently, there are no specific queries provided. Please check back later for updates.
```
Comment (filippoweb3) [inline docs/general/dune-analytics/parachain-dashboards/polkadex-dashboards.md]:
```suggestion
Currently, there are no specific queries provided.
```

File: docs/general/dune-analytics/parachain-dashboards/snowbridge-dashboards.md [hunk 45]
```diff
@@ -0,0 +1,86 @@
...
+title: Snowbridge on BridgeHub
+sidebar_label: Snowbridge
+description: Snowbridge is a general-purpose, trustless, and decentralized bridge that connects
+  Polkadot and Ethereum. This bridge facilitates communication between any Polkadot parachain and
+  Ethereum using Polkadot's XCMP messaging protocol. Parachains can send XCM instructions to
+  BridgeHub, leveraging Snowbridge for cross-chain interactions.
+
+  For more details on bridging with Kusama, please refer to our [BridgeHub
+  Dashboard](https://dune.com/substrate/bridgehub).
+
```
Comment (filippoweb3) [inline docs/general/dune-analytics/parachain-dashboards/snowbridge-dashboards.md]:
```suggestion
description: Snowbridge is a general-purpose, trustless, and decentralized bridge that connects
  Polkadot and Ethereum.
```

File: docs/general/dune-analytics/parachain-dashboards/snowbridge-dashboards.md [hunk 46]
```diff
@@ -0,0 +1,86 @@
...
+Ethereum. This bridge facilitates communication between any Polkadot parachain and Ethereum using
+Polkadot's XCMP messaging protocol. Parachains can send XCM instructions to BridgeHub, leveraging
+Snowbridge for cross-chain interactions.
+
+For more details on bridging with Kusama, please refer to our
+[BridgeHub Dashboard](https://dune.com/substrate/bridgehub).
+
+## Featured Dashboards on Dune
+
+Here you'll find a variety of dashboards that help visualize data from the Snowbridge protocol:
```
Comment (filippoweb3) [inline docs/general/dune-analytics/parachain-dashboards/snowbridge-dashboards.md]:
```suggestion
Here you will find a variety of dashboards that help visualize data from the Snowbridge protocol:
```

File: docs/general/dune-analytics/polkadot-ecosystem-overview.md [hunk 47]
```diff
@@ -0,0 +1,79 @@
+---
+id: polkadot-ecosystem-overview
+title: Polkadot Ecosystem Overview
+sidebar_label: Polkadot Ecosystem Overview
+description: High level metrics for Polkadot Ecosystem.
```
Comment (filippoweb3) [inline docs/general/dune-analytics/polkadot-ecosystem-overview.md]:
```suggestion
description: High-level metrics for Polkadot Ecosystem.
```

File: docs/general/dune-analytics/project-dashboards.md [hunk 48]
```diff
@@ -0,0 +1,26 @@
+---
+id: project-dashboards
+title: Project Dashboards
+sidebar_label: Project Dashboards
+description:
+  A list of dashboards about some of the interesting projects in the Polkadot ecosystem. The
+  dashboards are created using Dune Analytics and DuneSQL.
```
Comment (filippoweb3) [inline docs/general/dune-analytics/project-dashboards.md]:
```suggestion
  A list of dashboards about some of the interesting projects in the Polkadot ecosystem.
```

**Review by filippoweb3 — CHANGES_REQUESTED** [review]
Thanks for this contribution! Main points for the review:
- Page description must be as short as possible
- you'll -> you will, no abbreviations
- If there are no useful queries (most of the time it seems), consider removing that section
- Avoid "check back later" as this implies you will need to keep updated that part anytime soon

Feel free to shorten descriptions even further, as it seems the same info is present in the Overview section of each page.

Hope this helps! :)

**Comment by mkchungs** [comment]
Thanks for your feedback! We will improve the documentation and get back to you early next week!

**Comment by mkchungs** [comment]
@filippoweb3, we have incorporated the changes based on your feedback. We have also systematically added a few more useful queries (along with descriptions) for every parachain we cover. Hopefully, users will find these queries insightful and start building more awesome queries with the granular data on Dune.

Please take a look with the latest edit and let us know what else we can improve. Thanks

**Review by filippoweb3 — APPROVED** [review]
thank you for implementing the changes and adding more stuff. Nice to see some data on the Wiki :)

**Pull request merged**